src/main/include/subsystems/Flywheel.hpp

```diff
@@ -115,6 +115,9 @@ class Flywheel : public ControllerSubsystemBase {
     frc::Encoder m_encoder{Constants::Flywheel::kEncoderA,
                            Constants::Flywheel::kEncoderB};
     FlywheelController m_controller;
+    units::radian_t m_angle;
+    units::radian_t m_lastAngle;
+    std::chrono::steady_clock::time_point m_time;
     std::chrono::steady_clock::time_point m_lastTime =
         std::chrono::steady_clock::now();
```

Comment:
This will fix the time calculation when GetAngularVelocity() is called before ControllerPeriodic():
```suggestion
    std::chrono::steady_clock::time_point m_time =
        std::chrono::steady_clock::now();
    std::chrono::steady_clock::time_point m_lastTime =
        m_time - std::chrono::milliseconds(5);
```